### PR TITLE
Add instructions on how to use bookmarks into UG.

### PR DESCRIPTION
**Pull request by josephhhhhhhhh**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-    - [3.12. Displaying stats: `stats`](#312-displaying-stats-stats)
-    - [3.13. Exiting the program: `exit`](#313-exiting-the-program-exit)
-    - [3.14. Saving user data](#314-saving-user-data)
-  - [4. FAQ](#4-faq)
-  - [5. Command summary](#5-command-summary)
-  - [6. Glossary](#6-glossary)
-        - [1. Command Line Interface (CLI)](#1-command-line-interface-cli)
-        - [2. Object-oriented Programming (OOP)](#2-object-oriented-programming-oop)
+* Table of Contents
+{:toc}
```
Comment (kstonekuan) [inline docs/UserGuide.md:3]: cool change, no need to update this everytime anymore

File: docs/UserGuide.md [hunk 2]
```diff
@@ -236,7 +213,31 @@ Example of usage:
...
+Lists out all the bookmarked questions and each of their four options.
+
+Note:
+- Listing out all bookmarked questions can only be done in the main menu
+
+Example of usage:
+```
+bookmark
+```
+
```
Comment (kstonekuan) [inline docs/UserGuide.md:239]: would be good to add some screenshots here

**Review by kstonekuan — APPROVED** [review]
lgtm

File: docs/UserGuide.md (hunk 2 above)
Comment (josephhhhhhhhh) [reply docs/UserGuide.md:239]: Excellent idea, I shall get on it right away. Hold off on merging this branch.

**Pull request merged**